tests/FiberTraces.py

```diff
@@ -125,528 +125,519 @@ def testFiberTraceFunctionFindingControl(self):
-            fiberTrace.getTrace().getImage().getArray()[5,5] = val
-            self.assertAlmostEqual(fiberTraceMIF.getTrace().getImage().getArray()[5,5], val)
-
-            # Test setImage
-            val = 1011.
-            fiberTrace.getImage().getArray()[5,5] = val
-            self.assertTrue(fiberTraceMIF.setImage(fiberTrace.getImage()))
-            try:
-                fiberTraceMIF.setImage(maskedImageWrongSize.getImage())
-            except:
-                e = sys.exc_info()[1]
-                message = str.split(e.message, "\n")
-                expected = "FiberTrace.setImage: ERROR: image.getHeight(="+str(height)+") != _trace->getHeight(="+str(fiberTraceMIF.getHeight())+")"
-                self.assertEqual(message[0],expected)
-            self.assertAlmostEqual(fiberTraceMIF.getImage().getArray()[5,5], val)
-            fiberTrace.getImage().getArray()[5,5] = val+2
-            self.assertAlmostEqual(fiberTraceMIF.getImage().getArray()[5,5], val+2)
-
-            # Test setVariance
-            val = 1000.
-            fiberTrace.getVariance().getArray()[5,5] = val
-            self.assertTrue(fiberTraceMIF.setVariance(fiberTrace.getVariance()))
-            try:
-                fiberTraceMIF.setVariance(maskedImageWrongSize.getVariance())
-            except:
-                e = sys.exc_info()[1]
-                message = str.split(e.message, "\n")
-                expected = "FiberTrace.setVariance: ERROR: variance.getHeight(="+str(maskedImageWrongSize.getVariance().getHeight())+") != _trace->getHeight(="+str(fiberTraceMIF.getHeight())+")"
-                self.assertEqual(message[0],expected)
-            self.assertAlmostEqual(fiberTraceMIF.getVariance().getArray()[5,5], val)
-            fiberTrace.getVariance().getArray()[5,5] = val+2
-            self.assertAlmostEqual(fiberTraceMIF.getVariance().getArray()[5,5], val+2)
-
-            # Test setMask
-            val = 1
-            fiberTrace.getMask().getArray()[5,5] = val
-            self.assertTrue(fiberTraceMIF.setMask(fiberTrace.getMask()))
+            spectrum = fiberTrace.extractFromProfile()
+            recImage = fiberTrace.getReconstructed2DSpectrum(spectrum)
+            diff = fiberTrace.getTrace().getImage().getArray() - recImage.getArray()
+            meanDiff = np.mean(diff)
+            self.assertLess(np.absolute(meanDiff), 0.001)
+            stdDevDiff = np.std(diff)
+            self.assertLess(np.absolute(stdDevDiff), 0.001)
+
+        # Fit profile with MkSlitFunc
+        ftpfc = drpStella.FiberTraceProfileFittingControl()
+        fiberTraceSet = drpStella.findAndTraceAperturesF(self.flat.getMaskedImage(), self.ftffc)
+        for iTrace in range(0, fiberTraceSet.size()):
+            fiberTrace = fiberTraceSet.getFiberTrace(iTrace)
             try:
-                fiberTraceMIF.setMask(maskedImageWrongSize.getMask())
+                spectrum = fiberTrace.calcProfile()
```

Comment:
This looks like a substantive change, but maybe the diff is confused.

Are there any non-whitespace changes on this ticket?